substratevm/DEBUGINFO.md

```diff
@@ -0,0 +1,207 @@
+Using the ptototype debug info feature
+--------------------------------------
+
+To add debug info to a generated native image add flag
+-H:+GenerateDebugInfo to the native image command line.
+
+    $ javac Hello.java
+    $ mx native-image -H:+GenerateDebugInfo Hello
```

Comment:
GenerateDebugInfo is `HostedOptionKey<Integer>` which does not support the `-H:+GenerateDebugInfo` syntax.

Reply:
Yes, it's meant to be -H:GenerateDebugInfo=1

Reply:
Fixed